### DISCUSSION TITLE
Windows Explorer integration

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#134] [General]
What I really need is not installer version, but this. This is utter important for me. For example, Skylark doesn't have an installer, but it has the option to install/uninstall Windows Explorer integration built right into the UI of the application.

https://github.com/adonais/skylark/releases/download/20231207034256/skylark_x64-v4.0.9.7z

**Comment by SpartanJ** [comment]
> but it has the option to install/uninstall Windows Explorer integration built right into the UI of the application

What kind of integration do you need?

**Reply by ghost** [reply]
![Untitled](https://github.com/SpartanJ/ecode/assets/156915310/25a158a5-a918-4ec8-bab7-b06b3cefc0c7)

**Reply by SpartanJ** [reply]
Oh so you basically want File Associations or anything else?

**Reply by ghost** [reply]
File Associations is not as important to me as the right click context menu for files and folders.

**Reply by SpartanJ** [reply]
Both are reasonable requests that I agree would be nice to have. You can create a feature request if it's not present yet.
